Source/Core/Core/IOS/Network/Socket.cpp

```diff
@@ -712,24 +715,82 @@ void WiiSockMan::Update()
       socket_iter = WiiSockets.erase(socket_iter);
     }
   }
-  s32 ret = select(nfds, &read_fds, &write_fds, &except_fds, &t);
+  s32 ret = poll(ufds.data(), static_cast<u32>(ufds.size()), 0);
+  if (ret < 0)
+  {
+    ERROR_LOG(IOS_NET, "SO_POLL poll failed (%d)", GetNetErrorCode(ret, "Update", true));
```

Comment:
Could you move the GetNetErrorCode just above the ERROR_LOG, I don't want to take the risk of #3143 happenning again...
```C++
if (ret < 0)
{
  const s32 error_code = GetNetErrorCode(ret, "Update", true);
```